### PR TITLE
Add password checking on setting.

### PR DESCRIPTION
**Pull request by dracos**
Minimal length and comparison against common password list.

### CODE REVIEW

File: web/js/validation_rules.js [hunk 1]
```diff
@@ -1,5 +1,11 @@
     validation_rules = {
         title: { required: true },
         detail: { required: true },
-        update: { required: true }
+        update: { required: true },
+        password_register: {
+          remote: {
+            url: '/auth/common_password',
+            type: 'post'
```
Comment (struan) [inline web/js/validation_rules.js]: Why do only the common password bit here but not the length?

Comment (dracos) [reply web/js/validation_rules.js]: I was going to say the length is handled by fixmystreet.js in the addMethod, but that's then fixed whereas I made server-side cobrand specific didn't I, hmm.

**Comment by dracos** [comment]
"Should this not also handle updating passwords too?" oh yes!

File: web/js/validation_rules.js (hunk 1 above)
Comment (dracos) [reply web/js/validation_rules.js]: Will still do it client side as it's quicker, but will pass a variable from server to client.

**Pull request merged**